### PR TITLE
Use `mt_ ` random functions

### PR DESCRIPTION
**Pull request by lucasmichot**
Use `mt_ ` random functions

### CODE REVIEW

File: src/Carbon/Lang/fr.php [hunk 1]
```diff
@@ -102,7 +102,7 @@
             case 'd':
                 return $number.($number === 1 ? 'er' : 'e');
 
-            // Words with feminine grammatical gender: semaine
+                // Words with feminine grammatical gender: semaine
```
Comment (kylekatarnls) [inline src/Carbon/Lang/fr.php:105]:
```suggestion
            // Words with feminine grammatical gender: semaine
```

File: src/Carbon/Traits/Date.php [hunk 2]
```diff
@@ -913,102 +913,102 @@ public function get($name)
 
                 return is_numeric($value) ? (int) $value : $value;
 
-            // @property-read string long name of weekday translated according to Carbon locale, in english if no translation available for current language
+                // @property-read string long name of weekday translated according to Carbon locale, in english if no translation available for current language
```
Comment (kylekatarnls) [inline src/Carbon/Traits/Date.php:916]:
```suggestion
            // @property-read string long name of weekday translated according to Carbon locale, in english if no translation available for current language
```

File: src/Carbon/Traits/Date.php [hunk 3]
```diff
@@ -913,102 +913,102 @@ public function get($name)
...
             case $name === 'dayName':
                 return $this->getTranslatedDayName();
-            // @property-read string short name of weekday translated according to Carbon locale, in english if no translation available for current language
+                // @property-read string short name of weekday translated according to Carbon locale, in english if no translation available for current language
```
Comment (kylekatarnls) [inline src/Carbon/Traits/Date.php:919]:
```suggestion
            // @property-read string short name of weekday translated according to Carbon locale, in english if no translation available for current language
```

File: src/Carbon/Traits/Date.php [hunk 4]
```diff
@@ -913,102 +913,102 @@ public function get($name)
...
             case $name === 'shortDayName':
                 return $this->getTranslatedShortDayName();
-            // @property-read string very short name of weekday translated according to Carbon locale, in english if no translation available for current language
+                // @property-read string very short name of weekday translated according to Carbon locale, in english if no translation available for current language
```
Comment (kylekatarnls) [inline src/Carbon/Traits/Date.php:922]:
```suggestion
            // @property-read string very short name of weekday translated according to Carbon locale, in english if no translation available for current language
```

**Review by kylekatarnls — CHANGES_REQUESTED** [review]
Thanks.

All random functions are in tests. So I would expect to have only test folder touched in this PR.

At least, I would not touch comment indent in switch witch is actually wrong here.

File: src/Carbon/Lang/fr.php (hunk 1 above)
Comment (lucasmichot) [reply src/Carbon/Lang/fr.php:105]: Those are actually `php-cs-fixer` (`vendor/bin/php-cs-fixer fix`) fixes, if I revert those back, [the CS](https://github.com/briannesbitt/Carbon/blob/master/.github/workflows/phpcs.yml) workflow will fail.

In this case, we need to negate the `statement_indentation` rule

Comment (lucasmichot) [reply src/Carbon/Lang/fr.php]: cf. https://github.com/briannesbitt/Carbon/pull/2851/commits/3205687d0c7c6d1db5ce726942a8c7fe9c20ee0c

Comment (lucasmichot) [reply src/Carbon/Lang/fr.php]: I believe some changes might be related to an update of version of PHP-CS-Fixer

**Review by kylekatarnls — APPROVED** [review]

**Pull request closed without merging**